tests/browser_context_test.go

```diff
@@ -230,3 +232,33 @@ func TestBrowserContextUnrouteShouldWork(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, []int{4}, intercepted)
 }
+
+func TestBrowserContextShouldReturnBackgroundPage(t *testing.T) {
+	BeforeEach(t)
+	defer AfterEach(t)
+	if !isChromium {
+		t.Skip()
+	}
+	extensionPath := Asset("simple-extension")
+	context, err := browserType.LaunchPersistentContext(
+		t.TempDir(),
+		playwright.BrowserTypeLaunchPersistentContextOptions{
+			Headless: playwright.Bool(false),
+			Args: []string{
+				fmt.Sprintf("--disable-extensions-except=%s", extensionPath),
+				fmt.Sprintf("--load-extension=%s", extensionPath),
+			},
+		},
+	)
+	require.NoError(t, err)
+	defer context.Close()
+	var page playwright.Page
+	if len(context.BackgroundPages()) == 1 {
+		page = context.BackgroundPages()[0]
+	} else {
+		page = context.WaitForEvent("backgroundpage").(playwright.Page)
+	}
+	require.NotNil(t, page)
+	require.NotContains(t, context.Pages(), page)
+	require.Contains(t, context.BackgroundPages(), page)
+}
```

Comment:
close logic is here missing, that when its closed the background page gets removed.